frontend/src/constants/mapConstants.ts

```diff
@@ -0,0 +1,17 @@
+import { hansel } from "@/interfaces";
+
+export const SINGAPORE_CENTER: hansel.LatLng = Object.freeze({
+  lat: 1.3521,
+  lng: 103.8198,
+});
+
+export const DEFAULT_MAP_CONFIG: google.maps.MapOptions = Object.freeze({
```

Comment:
I'm getting this error here: cannot find namespace 'google'.ts(2503), I think we gotta add google.maps to tsconfig?

Reply:
:o does it work now

Reply:
Yea! Tqtq